.travis.yml

```diff
@@ -17,9 +17,9 @@ before_install:
 
 matrix:
   include:
-  - rvm: 1.9.3
-  - rvm: 2.0
-  - rvm: 2.2
+  - rvm: 2.2.5
```

Comment:
If we can't abandon 2.1, then we can't do this change at all. Note that Chef and Chef DK have abandoned 2.1 (as of the next release) and in fact will not `gem install` on those versions anymore without a `Gemfile`.